launcher/container_runner.go

```diff
@@ -508,22 +510,25 @@ func (r *ContainerRunner) Run(ctx context.Context) error {
 		defer teeServer.Shutdown(ctx)
 	}
 
-	// start node-problem-detector.service to collect memory related metrics.
-	if r.launchSpec.MemoryMonitoringEnabled {
-		r.logger.Println("MemoryMonitoring is enabled")
-		s, err := systemctl.New()
-		if err != nil {
-			return fmt.Errorf("failed to create systemctl client: %v", err)
-		}
-		defer s.Close()
+	if r.launchSpec.Experiments.EnableMemoryMonitoring {
+		r.logger.Println("EnableMemoryMonitoring experiment flag is enabled")
```

Comment:
nit: we log the whole launch spec already, we should remove these

Reply:
Done.